packages/perseus/src/widgets/interactive-graphs/mafs-styles.css

```diff
@@ -1,11 +1,12 @@
 /**
- * Overrides of mafs styles-- we will want to move this into JS land
- * to take advantage of WB tokens, but I cannot get that to work at the moment
+ * Overrides of mafs theme-- we will want to move this into JS land
```

Comment:
Would a good "half step" improvement be to make this a `.less` file and then at least be able to have Less variables? 

I think we already have some of our WB colours defined in [`variables.shared.less`](https://github.com/Khan/perseus/blob/c431c4b8147ae0630df2d2b19b0f5a5b5f04d4bf/packages/perseus/src/styles/variables.shared.less#L79)